docker-compose-prod.yml

```diff
@@ -0,0 +1,75 @@
+version: "3.8"
+
+# Add this file to extend the docker-compose setup, e.g.:
+# docker-compose -f docker-compose-prod.yml --env-file .env.prod build --no-cache
+# docker-compose -f docker-compose-prod.yml --env-file .env.prod up -d --build --force-recreate
+
+services:
+  app_prod:
+    build:
+      context: .
+      args:
+        ETHERPAD_PLUGINS: >-
+          ep_comments_page
+          ep_image_upload
+          kitsteam/ep_push2delete
+          ep_embedded_hyperlinks2
+          ep_headings2
+          ep_align
+          ep_font_color
+          ep_delete_empty_pads
+          ep_delete_after_delay
+          ep_helmet
+          ep_font_size
+          ep_disable_imports
+        INSTALL_SOFFICE: true
+    depends_on:
+      - mariadb_prod
+    environment:
+      DB_CHARSET: ${DOCKER_COMPOSE_APP_PROD_ENV_DB_CHARSET:-utf8mb4_bin}
+      DB_HOST: mariadb_prod
+      DB_NAME: ${DOCKER_COMPOSE_MARIADB_PROD_ENV_MARIADB_DATABASE:?}
+      DB_PASS: ${DOCKER_COMPOSE_MARIADB_PROD_ENV_MARIADB_PASSWORD:?}
+      DB_PORT: ${DOCKER_COMPOSE_MARIADB_PROD_ENV_MARIADB_PORT:-3306}
+      DB_TYPE: "mysql"
+      DB_USER: ${DOCKER_COMPOSE_MARIADB_PROD_ENV_MARIADB_USER:?}
+      DEFAULT_PAD_TEXT: ${DOCKER_COMPOSE_APP_PROD_ENV_DEFAULT_PAD_TEXT}
+      DISABLE_IP_LOGGING: ${DOCKER_COMPOSE_APP_PROD_ENV_DISABLE_IP_LOGGING:-true}
+      SOFFICE: ${DOCKER_COMPOSE_APP_PROD_ENV_SOFFICE:-/usr/bin/soffice}
+      TRUST_PROXY: ${DOCKER_COMPOSE_APP_PROD_ENV_TRUST_PROXY:-true}
+    ports:
+      - "${DOCKER_COMPOSE_APP_PROD_PORT_PUBLISHED:-9001}:${DOCKER_COMPOSE_APP_PROD_PORT_TARGET:-9001}"
+
+  mariadb_prod:
+    image: mariadb
+    # Pass config parameters to the postgres server.
+    # Find more information below when you need to generate the ssl-relevant file your self
+    command:
+      - --character-set-server=utf8mb4
```

Comment:
@tim-krieger Can you please investigate how your current database is setup in regards to the charset and the collation. This will help us to fix the error message in the log, see https://github.com/kitsteam/etherpad-lite/pull/1#issuecomment-1221624904

Reply:
> We need to look into this error message and check how to fix this
> 
> ```
> etherpad-lite-app_prod-1       | [2022-08-21 21:21:36.667] [ERROR] console - Database is not configured with charset utf8mb4_bin -- This may lead to crashes when certain characters are pasted in pads
> etherpad-lite-app_prod-1       | [2022-08-21 21:21:36.668] [INFO] console - {
> etherpad-lite-app_prod-1       |   DEFAULT_CHARACTER_SET_NAME: 'utf8mb4',
> etherpad-lite-app_prod-1       |   DEFAULT_COLLATION_NAME: 'utf8mb4_bin'
> etherpad-lite-app_prod-1       | } utf8mb4_bin
> etherpad-lite-app_prod-1       | [2022-08-21 21:21:36.673] [ERROR] console - table is not configured with charset utf8mb4_bin -- This may lead to crashes when certain characters are pasted in pads
> ```

I fixed this error message. I noticed that `DB_CHARSET` given was set to `utf8mb4_bin`. But, this is a collation server  setting and not a valid character set.